assets/src/realtime.d.ts

```diff
@@ -114,6 +114,7 @@ export interface Vehicle {
   operatorFirstName: string
   operatorLastName: string
   operatorLogonTime: Date | null
+  overloadOffset?: number | undefined
```

Comment:
If it's already called `overloadOffset?` I believe the question mark makes the `| undefined` redundant.